newsfragments/1488.feature

```diff
@@ -0,0 +1,2 @@
+Include small integer types from scitbx (see https://github.com/cctbx/cctbx_project/pull/533 for details)
```

Comment:
This is fine for a commit message, which has the context of the changes, but if you read this in isolation or in the changelog you have no idea what it refers to.
```suggestion
Added `support for small integer types <https://github.com/cctbx/cctbx_project/pull/533>`_ to DIALS flex arrays
```